### DISCUSSION TITLE
Missing month in total and shares production data

### DISCUSSION THREAD

**Discussion by ghost** [jobvk/Home-Assistant-Windcentrale#75] [Support] [answered]
Hi,
I get the following error
 [custom_components.windcentrale.sensor] The month 2 is missing in total production data
 [custom_components.windcentrale.sensor] The month 2 is missing in shares production data

I have shares in two different windmills. For one I get information the other one keeps showing 0. I wonder if I made a config mistake or if there is another issue.

Appreciate any help

Thanks,
Ronald

**Comment by jobvk** [comment, marked as answer]
You sure it was a error? and not a warning?

The problem is the first day of the week on monday there is nothing to look for in the week so it gives the warning and the sensor will update on tuesday around 14:00.

The same is for the month of the first day of the month there is no data to show because the sensor updates on the second day also around 14:00.

**Reply by ghost** [reply]
Oh I am not sure. But had it only for one of the mills I own shares of. And I am not seeing any data of one mill in the energy tab. Ps: error/warning disappeared
